article-list.html

```diff
@@ -41,6 +41,16 @@ <h2>Article title will go here</h2>
 				<div class="title">
 					<h2>Article title will go here</h2>
 				</div>
+
+				<div class="nav-buttons">

```

Comment:
The layout is the exact same as layout 1, but I incorporated the next and previous buttons as seen in layout 2 into this version.